nextcord/application_command.py

```diff
@@ -2136,6 +2142,9 @@ def get_payload(self, guild_id: Optional[int]) -> dict:
                 #  the default (True) to ensure payload parity for comparisons.
                 ret["dm_permission"] = True
 
+        if self.nsfw:
+            ret["nsfw"] = self.nsfw
```

Comment:
To prevent commands from needlessly resyncing, `ret["nsfw"]` must be set due to the payload Discord currently gives us.
```suggestion
        ret["nsfw"] = self.nsfw
```
If you want further details about how/why this is necessary, \@mention me in Discord.